train_wave.py

```diff
@@ -89,10 +86,9 @@ def check_for_done(process_queue):
     from train import run
 
     if "config" in parser_args:
-        cfg = load_config(parser_args.config)
-    elif "pickle" in parser_args:
-        with open(parser_args.pickle, "rb") as pickle_file:
-            cfg = dill.load(pickle_file)
+        cfg = load_config_py(parser_args.config)
```

Comment:
mark as deprecated? or should we keep it?
Actually, it is never used here, so should be good for removal even.